symfony/client/src/orangehrmPimPlugin/pages/dataImport/EmployeeDataImport.vue

```diff
@@ -0,0 +1,204 @@
+            >
+          </li>
+          <li>
+            <oxd-text class="orangehrm-information-card-text"
+              >Each import file should be configured for 100 records or
+              less</oxd-text
+            >
+          </li>
+          <li>
+            <oxd-text class="orangehrm-information-card-text"
+              >Multiple import files may be required</oxd-text
+            >
+          </li>
+          <li>
+            <oxd-text class="orangehrm-information-card-text"
+              >Sample CSV file:
+              <a class="download-link" href="#">
+                <span @click="onClickDownload" class="download-link"
+                  >Download</span
+                >
+              </a>
+            </oxd-text>
+          </li>
+        </ul>
+      </div>
+
+      <oxd-form :loading="isLoading" @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                type="file"
+                label="Select File"
+                buttonLabel="Browse"
+                v-model="attachment.attachment"
+                :rules="rules.attachment"
+                required
+              />
+              <oxd-text class="orangehrm-input-hint" tag="p"
+                >Accepts up to 1MB</oxd-text
+              >
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+
+        <oxd-divider />
+        <oxd-form-actions>
+          <required-text />
+          <oxd-button
+            class="orangehrm-left-space"
+            displayType="secondary"
+            label="Upload"
+            type="submit"
+          />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {APIService} from '@/core/util/services/api.service';
+
+let attachmentModel = {
+  attachment: null,
+};
+
+export default {
+  props: {
+    allowedFileTypes: {
+      type: Array,
+      required: true,
+    },
+  },
+
+  data() {
+    return {
+      isLoading: false,
+      attachment: {
+        ...attachmentModel,
+      },
+      rules: {
+        attachment: [
+          v => {
+            return v !== null || 'Required';
+          },
+          v =>
+            (v && v.size && v.size <= 1024 * 1024) ||
+            'Attachment size exceeded',
+          v =>
+            (v &&
+              this.allowedFileTypes.findIndex(item => item === v.type) > -1) ||
+            'File type not allowed',
+        ],
+      },
+    };
+  },
+  setup() {
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      `api/v2/pim/csvImport`,
+    );
+
+    return {
+      http,
+    };
+  },
+  methods: {
+    onSave() {
+      this.isLoading = true;
+      this.http
+        .create({
+          ...this.attachment,
+        })
+        .then(response => {
+          const importedRecords = response.data.meta.total;
+          this.updateModel();
```

Comment:
looks like the logic is this method is not resetting the component's `attachment` state. 
can remove this method and do in one line this as `this.attachment = { ...attachmentModel }`